### ISSUE TITLE
Remove More Topics causes home tab to not load

### ISSUE THREAD

**Issue by mynametoad** [dayanch96/YTLite#159]
### Describe the bug

```yaml
When enabling remove more topics in feed settings, home tab will not liad
```

### Steps to reproduce

```yaml
1. go to settings
2. go to feed
3. enable remove more topics
4. relaunch youtube
5. go to home tab

observe home tab will show scrolling loading indicators but never actually load; blank black screen after loading indicators scroll off screen. this can be resolved by disabling remove more topics and relaunching youtube.
```

### Post Requisites

- [X] I have checked that a similar issue has not been reported before.
- [X] Verified the relevance of the project and the version you are using.
- [X] Provided a detailed description of the problem and have filled in all required fields.
- [X] I did not answer truthfully to **ALL** the above checkboxes.
- [X] By submitting this issue, **I confirm that I have filled in all the necessary fields and complied with the guidelines. Failure to comply may result in the issue being closed without discussion, and repeated violations may lead to restrict account from further participation.**

**Comment by dayanch96** [comment]
Should be fixed now

**Comment by mynametoad** [comment]
@dayanch96 still occurring on 5.0.1.

**Comment by dayanch96** [comment]
How can I reproduce it? Just enable this option and re-open the app?

**Comment by mynametoad** [comment]
> How can I reproduce it? Just enable this option and re-open the app?

Yep. It is 100% repro for me doing this.

**Comment by dayanch96** [comment]
> > How can I reproduce it? Just enable this option and re-open the app?
> 
> Yep. It is 100% repro for me doing this.

Tried it on 5.0 and 5.0.1 and can scroll until the end of the feed

**Comment by mynametoad** [comment]
Heres video. Let me know if theres anything else I can get to help:

https://vimeo.com/1005244916?share=copy

**Comment by systox** [comment]
I can’t reproduce it either. 

if you go to settings -> YouTube plus -> YouTube preferences (at bottom) -> export preferences ->  then upload here i can give your specific settings a shot to see if the problem occurs on my devices.

**Comment by dayanch96** [comment]
> Heres video. Let me know if theres anything else I can get to help:
> 
> https://vimeo.com/1005244916?share=copy

Probably videos on your version has the same text that I check in posts, and it removes all videos, so there nothing to load.

**Comment by mynametoad** [comment]
[com.dvntm.ytlite.plist.txt](https://github.com/user-attachments/files/16831107/com.dvntm.ytlite.plist.txt)

had to rename this .txt to upload. let me onow if you can rename and still use this. otherwise i sill upload it some place and link

**Comment by systox** [comment]
> [com.dvntm.ytlite.plist.txt](https://github.com/user-attachments/files/16831107/com.dvntm.ytlite.plist.txt)
> 
> had to rename this .txt to upload. let me know if you can rename and still use this. otherwise i will upload it some place and link

Dang, still could not reproduce. 

I applied your plist, full restarted the app, and the home videos still loaded. Hope this helps the developer. 

Ip15pm
iOS 18 b6
YTplus only ipa 5.0.0

**Comment by mynametoad** [comment]
Only difference hw/sw wise between us is that I am on 18.1 beta 3, so its likely I am on some new UI for the home tab. Not sure what else I can provide.

**Comment by systox** [comment]
Did you build the Deb yourself or use the precompiled ipa from telegram?

**Comment by dayanch96** [comment]
Fixed in 5.0.2

**Comment by mynametoad** [comment]
Yep good to go on my end with 5.0.2. Thanks for sticking with this one!
